detox/test/integration/utils/simplistic-runner.js

```diff
@@ -3,8 +3,8 @@ const detox = require('detox/internals');
 async function main() {
   try {
     await detox.init();
-    await test1().catch(e => console.error(e));
-    await test2().catch(e => console.error(e));
+    await test1();
```

Comment:
Both `test1` and `test2` have nothing bad or specific, so they are expected to pass.

I decided to stop shooting own foot, and follow the Occam's razor principle.